action.yml

```
@@ -25,6 +25,14 @@ inputs:
   data-provider-config-urls:
     description: "A comma-separated list of YAML config file URLs for data providers, file format: ProviderClassName.yaml"
     required: false
+  git-user-name:
+    description: "The git user name used when performing the report commit"
+    required: false
+    default: Fosstars
```

```suggestion
    default: "Fosstars"
```
I know this may not matter much, but just to be on safe side.